PowerUp/app/src/main/res/values/keys.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="current_points_key">current_scenario_points</string>
```

Comment:
This is probably not a good place to store this key, as these xml files are for strings displayed in the application. Instead, you should create a Java class to store the keys and store each key as a public static final String